pkg/ipamclaimswebhook/podmutator.go

```diff
@@ -65,6 +67,15 @@ func (a *IPAMClaimsValet) Handle(ctx context.Context, request admission.Request)
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	log.Info("webhook handling event")
```

Comment:
Can you explain why did you move it before the "is pod VM" check on `webhook: Annotate with primary network ipamclaim` commit? 
it kinda demotes what you said you did in the  `webhook: Check first vm annotation` commit, cause now it no longer is the first thing :)